src/schema/stylelintrc.json

```
@@ -5844,6 +5153,9 @@
 				}
 			},
 			"additionalProperties": false
+		},
+		"validate": {
+			"type": "boolean"
 		}
```

```suggestion
		}
```

@ybiquitous What about not adding this for now?
i.e. punting stylelint/stylelint#8087 to 16.12.0

Supporting `configFile` is actually a can of worms.
i.e. it's not the only option that would need that fix

---

> What about not adding this for now?

Yes, that sounds good 👍🏼